File: P3/e_dados/aula1/conta.py
def criar_conta(numero, titulo, saldo, limite):
    conta = {"numero ": numero, "titulo ": titulo, "Saldo ": saldo, "limite ": limite}
    return conta

def depositar(conta, valor):
    conta['saldo'] += valor

def saca(conta, valor):
    conta['saldo'] -= valor
    
def sacar(conta, valor):
    conta['saldo'] -= valor

File: p2/pablo/Conta.java
class Conta {
    int saldo;

    public sacar(int valor) throws Excessos {
        if (saldo < valor){
            throw new Excessos()
        }
    }

    public void deposiatar(int valor) {
       saldo += depositar
       System.out.println("Depositor feito com sucesso ")
    }

    public retirar(int valor) {
        saldo -= retirar
        System.out.println('Retirqndo feita com sucesso ')
        
    }
    public String verificar() {
        System.out.println(saldo)
    }

} 